e2e/config/azure.go

```diff
@@ -20,15 +20,16 @@ import (
 )
 
 type AzureClient struct {
-	Core          *azcore.Client
-	VMSS          *armcompute.VirtualMachineScaleSetsClient
-	VMSSVM        *armcompute.VirtualMachineScaleSetVMsClient
-	VNet          *armnetwork.VirtualNetworksClient
-	Resource      *armresources.Client
-	ResourceGroup *armresources.ResourceGroupsClient
-	AKS           *armcontainerservice.ManagedClustersClient
-	SecurityGroup *armnetwork.SecurityGroupsClient
-	Subnet        *armnetwork.SubnetsClient
+	Core                      *azcore.Client
+	VMSS                      *armcompute.VirtualMachineScaleSetsClient
+	VMSSVM                    *armcompute.VirtualMachineScaleSetVMsClient
+	VNet                      *armnetwork.VirtualNetworksClient
+	Resource                  *armresources.Client
+	ResourceGroup             *armresources.ResourceGroupsClient
+	AKS                       *armcontainerservice.ManagedClustersClient
+	SecurityGroup             *armnetwork.SecurityGroupsClient
+	Subnet                    *armnetwork.SubnetsClient
+	GalleryImageVersionClient *armcompute.GalleryImageVersionsClient
```

Comment:
where are we actually using this client? I see we also create one within `findLatestImageWithTag`, which seems to be the only place we're using any object of this type

Reply:
yep, this is the only usage of the client